templates/blocks/quote.html

```diff
@@ -1,23 +1,18 @@
 {% load static %}
 
 <div class="quote">
-    <div class="container">
-        <img class="quote__icon" src="{% static 'images/fontawesome-svgs/quote-left.svg' %}" alt="" />
-        {% if value.heading %}
-            <h3 class="sr-only">{{ value.heading }}</h3>
-        {% endif %}
```

Comment:
Note: Are you sure we no longer need the icon?

Reply:
I think as the styles are changing anyways - it shouldn't matter, however in the one that was actually being used (`articles/blocks/quote.html`) there was no icon :)